elements/pf-popover/pf-popover.ts

```diff
@@ -306,9 +306,9 @@ export class PfPopover extends LitElement {
     const hasIcon = this.#slots.hasSlotted('icon') || !!this.icon || !!this.alertSeverity;
     const fallbackIcon = this.icon ?? (this.alertSeverity ? alertIcons[this.alertSeverity] : nothing);
 
-    const screenReaderText = this.alertSeverity ?
-      html`<span class="sr-only">${this.alertSeverityText ?? `${this.alertSeverity} alert:`}</span>`
-      : nothing;
+    const screenReaderText = !this.alertSeverity ? nothing : html`
```

Comment:
@bennypowers saw a similar change in a previous commit...why are we flipping the condition around? TI(will)L? :)

Reply:
it's nice to have the big html chunk after wards, because then there's no trailing "or empty" afterwards.

in other words, since in cases like this we mean to say "only show this template if", it makes sense to put all the sad-path stuff together